docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -361,8 +360,6 @@ This subsection is **only** for repositories under `ansible-collections <https:/
 
 All new repositories MUST have ``main`` as the default branch.
 
-Existing repositories SHOULD be converted to use ``main``.
-
```

Comment:
Why remove this?

Reply:
> Why remove this?

@briantist this was relevant after that decision to convert was made and we converted where we could and created issues in **all** other repos under ansible-collections. Who wanted already did it and new repos are created there with `main`. Maybe we should rephrase it somehow